db/migrate/20230319115620_move_rss_yaml_to_db.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class MoveRssYamlToDb < ActiveRecord::Migration[7.0]
+  def up
+    feeds = YAML.safe_load(SiteSetting.rss_polling_feed_setting)
```

Comment:
Is it possible for this setting to be invalid/empty? (we wouldn't like for the migration to fail in that case)